### ISSUE TITLE
Image - a full research proposal or just a definition proposal?

### ISSUE THREAD

**Issue by kubkon** [WICG/open-ui#278]
Similarly to #257, thanks to a great shoutout by @gregwhitworth, I think it'd be useful to gather some feedback and opinions regarding the image component. The question is, should image have a full research proposal or would just a definition proposal be enough? You can find some prelim research on the topic in #274, and when you browse through it, you'll notice that quite a few design systems don't really have a separate "image" component, instead relying on the `<img/>` tag with some dedicated CSS classes (which for instance is the case for [bootstrap](https://getbootstrap.com/docs/4.0/content/images/#responsive-images) -- I've included it in the research PR to showcase the approach).

**Comment by levithomason** [comment]
Open UI has the responsibility to document UI as it has naturally evolved in the industry (descriptivism). Because there are many design systems and libraries which describe a UI component called `Image`, we then have the responsibility to document and research it. We should also capture the concepts those systems have associated to image.

Components which are not common in reputable design systems and component libraries will not be documented. Image and Link do not fall into this category.

**Comment by levithomason** [comment]
Quoting my own reply and @gregwhitworth's response here for completeness (from #274):

>>**(Levi)** @gregwhitworth / @una Yes, as Open UI has the responsibility to descriptively document UI as it has naturally evolved in the industry we should then have a research page for any component scope that exists in reputable sources. As there is an Image component in many design systems and component libraries, we should then have an associated research page that documents that component.
>
>**(Greg)** @levithomason I have no opposition to research pages, not sure who does but there does need to be an expected next step for those research pages. We don't expect every component to make its way to the platform but we do expect there to be a landing spot. And as we noted for card it may solely be a definition page. This comes full circle to information architecture for Open UI as all research/proposals are given the same weight and implied direction with many developers having excitement at the proposition of new elements. We need to ensure that the site clearly articulates this.
>
>Additionally, specific to an Image component the research should go beyond the imagery and understanding why the component libraries/design systems have them as there may be possible platform implications or a need beyond a definition style page. And as a group there should be a resolution on what level of specification it aligns with. Again, we're getting a bit more into process weeds here but I don't think anyone is opposed to research in general - it's the next step that we're interested in and clarity in labelling etc will help with this.

**Comment by gregwhitworth** [comment]
@levithomason @kubkon this was discussed briefly today during the information architecture and I referenced `<image>`, `<skeleton>`, `<card>`, `<link>` and others that deserve a place for reference. There will also be an effort to try and address the multiple different types of personas (possibly by the same people just different contexts) that work with this content. There have been discord channels and there will be sub-group working sessions to work out how best to ensure that all of the goals that Open UI is out to tackle are clear, discoverable, and consumable. The items prefixed with `ia` in the discord channel are the different work groups.

**Comment by levithomason** [comment]
This is fantastic. I also scanned the Google doc on the topic. I really like the direction there. I also gave some feedback on the homepage redesign. I really like the idea of separating the analysis from the proposals at the top-level.

Thanks much for ensuring the research side stays in tact here. I do truly believe this is one of the keys that has been missing in this kind of effort and is essential to ensuring platform iterations meet the needs of the real-world in the long term.
